eng/Version.Details.xml

```diff
@@ -27,5 +27,9 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>db49d790a4bfa977a9ed7436bf2aa242cefae45e</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.TemplateEngine.Tasks" Version="7.0.100-alpha.1.21601.1">
+      <Uri>https://github.com/dotnet/templating</Uri>
+      <Sha />
```

Comment:
Is the lack of hash here expected?

Reply:
I have taken this part from the earlier PR in dotnet/maui repo. https://github.com/dotnet/maui/pull/4439/files
Seems to be OK not to have a hash.